test/Bridges/flip_sign_bridge.jl

```diff
@@ -10,7 +10,8 @@ include("utilities.jl")
 
 include("simple_model.jl")
 
-mock = MOIU.MockOptimizer(SimpleModel{Float64}())
+# Unit tests need to set `MOI.Silent` so we need `MOIU.UniversalFallback`
```

Comment:
Nothing to address in this PR, but it's pretty weird requiring an additional new layer just for one attribute.